src/main/java/org/isf/dlvrtype/gui/DeliveryTypeBrowserEdit.java

```diff
@@ -215,7 +216,9 @@ public void actionPerformed(java.awt.event.ActionEvent e) {
                             if (result) {
                                 fireDeliveryInserted();
                             }
-                            if (!result) JOptionPane.showMessageDialog(null, MessageBundle.getMessage("angal.dlvrtype.thdatacouldnotbesaved"));
+                            if (!result) {
+                                MessageDialog.error(null, "angal.dlvrtype.thdatacouldnotbesaved");
```

Comment:
reminder: we could move this bundle to `common` key

Reply:
Yes; there are a few others like that too.  In this pass I didn't want to touch any of the keys but I will do that soon.